src/redux/apiFunctions.js

```diff
@@ -31,4 +32,19 @@ const loadMissions = async () => {
   return missionsObj;
 };
 
-export { loadMissions, loadRockets };
+const loadDragons = async () => {
+  const respond = await fetch(dragonData).then((res) => res.json()).then((GetDragons) => GetDragons);
+  const dragonsItems = respond.map((dragon) => (
+    {
+      id: dragon.dragon_id,
+      name: dragon.dragon_name,
+      description: dragon.description,
+      image: dragon.flickr_images[0],
```

Comment:
[REQUIRED] Please, do well to check your API and fetch correctly. Replace description with type, and use the appropriate property keys as offered by the API.